resources/views/volumes/videos/index/meta.blade.php

```diff
@@ -0,0 +1,51 @@
+<div class="panel panel-default">
+    <div class="panel-heading">
+        <h3 class="panel-title">Video information</h3>
+    </div>
+    <div class="panel-body table-responsive">
+        <table class="table">
+            @if ($video->width && $video->height)
+                <tr>
+                    <th>Dimensions</th>
+                    <td>{{ $video->width }} &times; {{ $video->height }} px </td>
+                </tr>
+            @endif
+            @if ($video->size)
+                <tr>
+                    <th>Size</th>
+                    <td>{{ round($video->size / 1E+6, 2) }} MBytes </td>
+                </tr>
+            @endif
+            @if ($video->mimetype)
+                <tr>
+                    <th>MIME</th>
+                    <td><code>{{ $video->mimetype }}</code></td>
+                </tr>
+            @endif
+            @if ($video->taken_at)
+                <tr>
+                    <th>Created</th>
+                    @if (is_array($video->taken_at))
+                        @foreach ($video->taken_at as $value)
+                            <td>{{ $value }}</td>
+                        @endforeach
+                    @else
+                        <td>{{ $video->taken_at }}</td>
+                    @endif
+                </tr>
+            @endif
+            @foreach ($metadata as $key => $field)
+                <tr>
+                    <th>{{ $metadataMap[$key] }}</th>
+                    @if (is_array($field))
+                        @foreach ($field as $value)
+                            <td>{{ $value }}</td>
+                        @endforeach
+                    @else
+                        <td>{{ $value }}</td>
+                    @endif
+                </tr>
+            @endforeach
```

Comment:
Video metadata may be an array, too. This will produce an "array to string conversion" error here. How could we show these values here in a meaningful way? Maybe show the min/max values of each field only?

Reply:
That is why I added the `@if(is_array)` loop for `taken_at` and each metadata field. I tested with attached metadata file as array and it works. The result is quite heavy so yes, maybe we could print only the min/max or first/last values, what do you think ?

![Capture](https://github.com/user-attachments/assets/f7525198-a217-4e36-9dca-3997543a532e)

[biigle-volume-6-metadata.csv](https://github.com/user-attachments/files/18276307/biigle-volume-6-metadata.csv)